_data/simple-icons.json

```
@@ -5479,6 +5479,11 @@
             "source": "https://www.atlassian.com/company/news/press-kit",
             "guidelines": "https://atlassian.design/foundations/logos/"
         },
+        {
+            "title": "JitPack",
+            "hex": "FFFFFF",
+            "source": "https://jitpack.io/w/img/jitpack-logo.svg"
```

From where exactly did you extract this URL? The icon in the hero section of [jitpack.io](https://jitpack.io) is https://jitpack.io/w/img/jitpack_bg_opacity01.png. I found your provided source URL on [docs.jitpack.io](https://docs.jitpack.io/). Is this it?

According to the [source guidelines of the contribution guide](https://github.com/simple-icons/simple-icons/blob/d4dd8016f0f8cf92a6aa6b92e686b7c9889bc378/CONTRIBUTING.md#source-guidelines):

> If the SVG is sourced from:
> ...
> * **Company website:** If the SVG is found on the company website (but there is no branding page) the source URL should link to a common page, such as the home page or about page, that includes the source image and not the image file itself.
> ...

For this reason, please update the source URL to the website on which includes the source image instead of using the URL of the image file. If the website from which you retrieved the source image is [docs.jitpack.io](https://docs.jitpack.io/), then this would be the required change:

```suggestion
            "source": "https://docs.jitpack.io"
```